server/controllers/loginController.test.js

```diff
@@ -15,17 +15,15 @@ test('login with email and password', t => {
   }
   const UserModel = {
     findOne (query, fieldsToReturn, cb) {
-      if (query.email === req.body.email && fieldsToReturn === 'email password') {
+      if (query.email === req.body.email && fieldsToReturn === 'email password opportunitiesAppliedFor') {
         cb(null, req.body)
       }
     }
   }
 
   // mocks
   const resMock = mock(res)
-    .expects('sendStatus')
-    .once()
-    .withExactArgs(200)
+    .expects('send')
```

Comment:
Add the following to the top and remove the `const res = ...` in all tests for both the `loginController.test.js` and `signupController.test.js` files.

```js
let res

test.beforeEach(() => {
  res = {
    sendStatus () {},
    setHeader () {},
    send () {}
  }
})
```

Also, modify the `signupController.test.js` when the `res` is mocked for the `signup new user` test to be:
```js
const mockRes = mock(res)
  .expects('send')
  .once()
  .withExactArgs(JSON.stringify({ savedUser: { email: 'user@email.com' } }))
```